tcms/templates/run/get_case_runs.html

```diff
@@ -156,20 +156,15 @@
 				{{/each}}
 			<div class="bug_type" id="id_bug_type">Issue Tracker</div>
 			<select name="bug_system_id" id="bug_system_id">
-				<option value="1" selected="selected" data-tab="bug-system-bugzilla">Bugzilla</option>
-				<option value="2" data-tab="bug-system-jira">JIRA</option>
+				{{#each bugTrackers}}
+					<option value="{{ @key }}">{{ this }}</option>
+				{{/each}}
 			</select>
 			</br>
-			<div id="bug-system-bugzilla" style="line-height:30px;">
+			<div id="bug-system" style="line-height:30px;">
 				<div class="bug_id">Bug ID</div>
 					<input type="text" name="bug_id">
 					</br>
-					<input type="checkbox" name="bz_external_track" value="bz_external_track">&nbsp;
-					<span class="use_bz_external_track">Check to add Test Cases to BZ</span>
```

Comment:
Scratch the part about tabs and UI, this is completely different ball game. We're not there yet. 

Adding back the checkbox is part of #185. I'm not going to include it in this PR.

You keep saying we shouldn't break existing functionality but the fact is that it is already broken even without this pull request. The hard-coded Bugzilla and JIRA related code works only for the Bugzilla and JIRA instances that are hard-coded. Given that by default we have  bugzilla.redhat.com  means nobody else besides Red Hat uses this setting because it is broken for them. More over without custom bug tracker support the entire Add/Remove bug is broken for everyone outside Red Hat.